### DISCUSSION TITLE
SolidJS examples?

### DISCUSSION THREAD

**Discussion by Sashley** [TanStack/virtual#631] [General]
Just asking if anyone has been working on examples for SolidJS, more specifically sorting within Tanstack Table and are willing to share any of the code? (github sources look significantly different, and don’t want to do deep dives and reinvent the wheel). TIA.

**Comment by SilentRhetoric** [comment]
Hi, seconding the need for SolidJS examples ❤️

**Reply by Sashley** [reply]
I'm a little embarrassed because it is still underdone, but was working on the following with tanstack table/ virtual. Some components are working, others need work (highlighted). Will expand this to include tanstack query. Due to get back it it shortly. https://github.com/Sashley/SolidJS-Tanstack-Playground
